person.rb

```diff
@@ -29,6 +30,10 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(rental)
```

Comment:
- [ ] It would be better if we pass the person itself as a parameter to create the new rental achieving the `has-many` association. For example;
```
def add_rental(book, date)
    Rental.new(date, book, self)
end
```
The `self` keyword helps us pass the person as a parameter in creating a new rental